packages/flutter/lib/src/painting/text_style.dart

```diff
@@ -122,6 +122,79 @@ import 'basic_types.dart';
 /// )
 /// ```
 ///
+/// ### Custom Fonts
+///
+/// Custom fonts can be declared in the `pubspec.yaml` file as shown below:
+///
+///```yaml
+/// flutter:
+///   fonts:
+///     - family: Raleway
+///       fonts:
+///         - asset: assets/fonts/Raleway-Regular.ttf
+///         - asset: assets/fonts/Raleway-Medium.ttf
+///           weight: 500
+///         - asset: assets/fonts/Raleway-SemiBold.ttf
+///           weight: 600
+///      - family: Schyler
+///        fonts:
+///          - asset: fonts/Schyler-Regular.ttf
+///          - asset: fonts/Schyler-Italic.ttf
+///            style: italic
+///```
+///
+/// The `family` determines the name of the font, which you can use in the
```

Comment:
For both `family` and `assert` either add `property` after it, or remove the proceeding `the`